src/utils/clone.ts

```diff
@@ -0,0 +1,3 @@
+import rfdc from 'rfdc'
```

Comment:
Why do this instead of simply using `rfdc`? To be able to add types?

Reply:
The `rfdc` module exports a function by default that you can pass configuration details to. So instead of having to re-create the default `clone` function each time, we can just export it from this module. And if we end up needing to change the configuration of how the clone function works, we'll be able to do it in just one place.